js/bootstrap-slider.js

```diff
@@ -984,20 +984,15 @@
 					var minTickValue = Math.min.apply(Math, this.options.ticks);
 
 					var styleSize = this.options.orientation === 'vertical' ? 'height' : 'width';
-					var styleMargin = this.options.orientation === 'vertical' ? 'marginTop' : 'marginLeft';
```

Comment:
why is this code being removed? how does this affect the orientation when the slider is vertical instead of horizontal?

id like to see a fiddle for this